assets/scss/index.scss

```diff
@@ -21,6 +21,7 @@ body {
     margin: 0;
     height: 100%;
     width: 100%;
```

Comment:
Talked with Line about this, people have complained about a "messy look" and "messy background picture"